openghg/standardise/emissions/_intem.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Dict, Literal, Optional, Union
+
+
+def parse_intem(
+    filepath: Path,
+    species: str,
+    source: str = "intem",
+    domain: str = "europe",
+    data_type: str = "emissions",
+    database: Optional[str] = None,
```

Comment:
Do we have a database version for INTEM @rt17603 ?

Reply:
INTEM isn't a database, it's an inversion model (run by Alistair and the met office).

For "emissions" we can expect them to come from different places (and can use different connected labels for this) e.g.
 - an output from a model run e.g. to simulate wetlands model (e.g. model)
 - bottom up inventory (e.g. database, database version etc.)
 - a posterior output from an inversion model (e.g. could use model again or could add a new input option)